packages/common/src/schemas/sign-on/selectGenresSchema.ts

```diff
@@ -1,5 +1,23 @@
 import { z } from 'zod'
 
+import { GENRES, Genre, convertGenreLabelToValue } from 'utils/genres'
+
+const excludedGenres = new Set<string>([
+  Genre.COMEDY,
+  Genre.KIDS,
+  Genre.SOUNDTRACK,
+  Genre.DEVOTIONAL,
+  Genre.AUDIOBOOKS,
+  Genre.SPOKEN_WORK
```

Comment:
man we're gonna miss out on all those people coming to Audius solely to hear their favorite slam poet, darn